guides/doc-Provisioning_Guide/topics/Introduction.adoc

```diff
@@ -22,24 +22,43 @@ Virtualization Infrastructure::
-. The DHCP server responds to the lease request and returns TFTP `next-server` and `filename` options.
-. The host requests the bootloader and menu from the TFTP server.
-. The PXELinux menu and OS installer for the host is returned over TFTP.
-. The installer requests the provisioning template or script from {Project}.
-. {Project} renders the template and returns the resulting kickstart file to the host.
-. The host enters a build process that installs the operating system, registers the host to {Project}, and installs management tools such as `katello-agent` and `puppet`.
-. The installer notifies {Project} of a successful build in the `postinstall` script.
-. The PXELinux menu reverts to a local boot template.
-. The host starts its operating system. If you configured the host to use any Puppet classes, the host configures itself using the modules stored on {Project}.
+* A PXELinux, Grub, Grub2 and iPXE configuration files are created for the host in the TFTP {SmartProxyServer} that is associated with the subnet.
+* A puppet certificate is generated on associated puppet server.
+* A realm is created on associated identity server.
+. The new host requests a DHCP reservation from the DHCP server.
+. The DHCP server responds to the reservation request and returns TFTP `next-server` and `filename` options.
+. The host requests the bootloader and menu from the TFTP server according to the PXELoader setting.
+. A boot loader is returned over TFTP.
+. The boot loader fetches configuration for the host via it's provision interface MAC address.
+. The boot loader fetches operating system installer kernel, init RAM disk and boot parameters.
+. The installer requests the provisioning template from {Project}.
+. {Project} renders the provision template and returns the result to the host.
+. The installer performs installation of the operating system.
+ifeval::["{build}" == "foreman"]
+* When Katello plugin is installed, the installer registers the host to {Project} using Red Hat Subscription Manager.
+* When Katello plugin is installed, the installer installs management tools such as `katello-agent` and `puppet`.
+endif::[]
+ifeval::["{build}" == "satellite"]
+* The installer registers the host to {Project} using Red Hat Subscription Manager.
+* The installer installs management tools such as `katello-agent` and `puppet`.
+endif::[]
+* The installer notifies {Project} of a successful build in the `postinstall` script.
+. The PXE configuration files revert to a local boot template.
+. The host reboots.
+. The new host requests a DHCP reservation from the DHCP server.
+. The DHCP server responds to the reservation request and returns TFTP `next-server` and `filename` options.
+. The host requests the bootloader and menu from the TFTP server according to the PXELoader setting.
+. A boot loader is returned over TFTP.
+. The boot loader fetches configuration for the host via it's provision interface MAC address.
```

Comment:
provisioning interface? provisioned interface?

Reply:
Sorry what? The interface that has "provision" flag turned on. You pick the correct term please.